cmd/snapshotgc/snapshotgc.go

```diff
@@ -122,3 +123,46 @@ func getUnassociatedNSSnapshots(
 
 	return unAssociatedSnaps, nil
 }
+
+// Keep a certain amount of pr/non-pr snapshots
+func getSnapshotsForRemoval(
+	cl client.Client,
+	snapshots []applicationapiv1alpha1.Snapshot,
+	prSnapshotsToKeep int,
```

Comment:
I must be overlooking something, but where are these filled with data? Both 
**prSnapshotsToKeep** and **nonPrSnapshotsToKeep**. I suppose in different PR, or in different part of the code?

Reply:
This will come from command line arguments passed to the main file. That will be in a PR coming once we merge the remaining two PRs, currently in review.

It's ready, but it required some stubbing, so I didn't want to open a PR for that as it might have been a bit confusing for review.

You can see it here: https://github.com/yftacherzog/integration-service/commit/baaea02c2002db7af87b480301b91ddcbc946bc1